.github/workflows/update-openmrs-deps.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  workflow_dispatch:
+  schedule:
+    # every hour
+    - cron: '30 */1 * * *'
```

Comment:
Shall we change this to daily or something?

Reply:
I figured this is relatively low-cost and it's probably better to get these updates in faster. Ideally, we'd just trigger this workflow from esm-core or something.